src/utils.ts

```diff
@@ -1,26 +1,28 @@
 import { ReadStream } from 'fs';
-import { Actor, Member, TaskRunner } from 'graasp';
+import path from 'path';
+
+import { Actor, Member, TaskRunner } from '@graasp/sdk';
 import { FileTaskManager } from 'graasp-plugin-file';
 import { DownloadFileInputType } from 'graasp-plugin-file/dist/tasks/download-file-task';
-import path from 'path';
+
 import { ItemValidationStatuses } from './constants';
 
-export const stripHtml = (str: string) => str?.replace(/<[^>]*>?/gm, '');
+export const stripHtml = (str: string): string => str?.replace(/<[^>]*>?/gm, '');
```

Comment:
This seems [extremly unsafe](https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags?noredirect=1&lq=1). HTML is not a regular language and cannot be parsed with regexes, and thus requires a finite automata to be processed correctly. Consider using https://www.npmjs.com/package/striptags instead.